src/applications/mhv-secure-messaging/containers/EditContactList.jsx

```diff
@@ -64,20 +65,36 @@ const EditContactList = () => {
     [allTriageTeams],
   );
 
-  const updatePreferredTeam = (triageTeamId, selected) => {
-    setAllTriageTeams(prevTeams =>
-      prevTeams.map(
-        team =>
-          team.triageTeamId === triageTeamId
-            ? {
-                ...team,
-                preferredTeam: selected || !team.preferredTeam,
-              }
-            : team,
-      ),
+  const SetStationCount = triageTeams => {
+    const teams = triageTeams || [];
+    const stationNumbers = teams.map(team => team.stationNumber);
+    const uniqueStationNumbers = [...new Set(stationNumbers)];
+    return Object.fromEntries(
+      uniqueStationNumbers.map(station => [
+        station,
+        teams.filter(
+          team => team.stationNumber === station && team.preferredTeam,
+        ).length,
+      ]),

```

Comment:
1. this function need to account for blocked teams and exclude them from count, otherwise wrong count may be displayed. Can we include this in test coverage?
2. [nitpick] const name should start with lower case